.circleci/config.yml

```diff
@@ -981,6 +985,22 @@ jobs:
       - store_test_results:
           path: test/test-results/e2e
 
+  test-e2e-chrome-vault-decryption:
+    executor: node-browsers-large
+    parallelism: 24
```

Comment:
Nit: We might not need 24 jobs to run in parallel here, given that we only have 1 test

Reply:
removed in 412d759b7d0132dbc91cbce3748936a22b776994